pkg/rtc/participant.go

```diff
@@ -1466,9 +1467,7 @@ func (p *ParticipantImpl) mediaTrackReceived(track *webrtc.TrackRemote, rtpRecei
 	if p.twcc == nil {
 		p.twcc = twcc.NewTransportWideCCResponder(ssrc)
 		p.twcc.OnFeedback(func(pkt rtcp.RawPacket) {
-			if err := p.publisher.pc.WriteRTCP([]rtcp.Packet{&pkt}); err != nil {
-				p.params.Logger.Errorw("could not write RTCP to participant", err)
-			}
+			p.rtcpCh <- []rtcp.Packet{&pkt}
```

Comment:
Posting this to channel. This is coming from a callback in twcc responder with a lock held and goes all the way into WebRTC land. Guess it was all okay. But, we have a RTCP worker and let's use that channel to post this message and let the worker handle it.